tests/foreman/cli/test_discoveryrule.py

```diff
@@ -172,15 +171,34 @@ def test_positive_create_with_org_loc_id(
         assert class_org.name in rule.organizations
         assert class_location.name in rule.locations
 
+        new_org = Box(target_sat.cli_factory.make_org())
+        new_loc = Box(target_sat.cli_factory.make_location())
```

Comment:
cli_factory methods returns Box object already
```suggestion
        new_org = target_sat.cli_factory.make_org()
        new_loc = target_sat.cli_factory.make_location()
```

Reply:
Good catch!